libs/ui-toolkit/src/components/ag-grid/ag-grid-lazy-themed.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import type { AgGridReactProps, AgReactUiProps } from 'ag-grid-react';
+import { AgGridReact } from 'ag-grid-react';
+import { ThemeContext } from '@vegaprotocol/react-helpers';
+import 'ag-grid-community/dist/styles/ag-grid.css';
+
+const AgGridLightTheme = React.lazy(() =>
+  import('./ag-grid-light').then((module) => ({
+    default: module.AgGrid,
+  }))
+);
+
+const AgGridDarkTheme = React.lazy(() =>
+  import('./ag-grid-dark').then((module) => ({
```

Comment:
does this add prefetch to the resource? I cannot remember if we need to configure something wiht webpack comments in order to force that. Would like to load in the background during idle time if possible